eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -18,6 +18,7 @@ Param (
   [string] $TargetBranch = ("origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" -replace "refs/heads/")
 )
 
+Set-StrictMode -version 3
```

Comment:
Make sure you do some validation after you enable this as it can cause your script to start failing in various ways.